build/lib/train.py

```
@@ -0,0 +1,271 @@
+        steps = 0
+        total_reward_left = 0.0
+        total_reward_right = 0.0
+
+        # Track average loss over the episode
+        loss_sum_left = 0.0
+        loss_sum_right = 0.0
+        loss_count_left = 0
+        loss_count_right = 0
+
+        last_info: Dict[str, Any] = {}
+        with record_pygame(f"videos/replay_hockey_{self.episode}.mp4", fps=60) as rec:
```

Ruff (0.12.2)

Local variable `rec` is assigned to but never used

Remove assignment to unused variable `rec`

(F841)